src/components/FacilitiesPageViewMore/DeleteFacility.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+    useDisclosure,
+    Button,
+    ButtonGroup,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalCloseButton,
+    ModalBody,
+    ModalFooter,
+} from '@chakra-ui/react';
+import { useBackend } from '../../contexts/BackendContext';
+
+
+function DeleteFacility(props) {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const { backend } = useBackend();
+    function ShowModal({ isOpen, onClose }) {
+        return (
+          <Modal isOpen={isOpen} onClose={onClose}>
+            <ModalOverlay />
+            <ModalContent>
+              <ModalHeader>Remove Facility</ModalHeader>
+              <ModalCloseButton />
+              <ModalBody>
+                Are you sure you want to remove the facility from the adoption log? Once you
```

Comment:
🚫 **[eslint]** <[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Replace `················Are·you·sure·you·want·to·remove·the·facility·from·the·adoption·log?·Once·you` with `············Are·you·sure·you·want·to·remove·the·facility·from·the·adoption·log?·Once·you·delete`
```suggestion
            Are you sure you want to remove the facility from the adoption log? Once you delete
```